DMTR-311.tex

```diff
@@ -40,8 +48,1000 @@
 
 \maketitle
 
-% ADD CONTENT HERE
-% You can also use the \input command to include several content files.
+In this Report, we characterize the performance of the Rubin Observatory Science Pipelines Version 22.0.0. We illustrate the performance via metrics that are measured on the HSC-RC2 dataset. RC2 consists of 3 tracts of data taken from the HSC-SSP survey, and selected to provide a means of testing various ``pathological'' cases (e.g., difficult astrometric solutions, seeing too small to provide a well-sampled PSF, difficult fields for deblending, and large galaxies, among others). These three tracts each contain between 112--149 visits split between the HSC-G, HSC-R, HSC-I, HSC-Z, and HSC-Y (\emph{grizy}) filters. 
+
+Beginning in March 2021, the metrics calculation package \texttt{validate\_drp} was deprecated and replaced with \texttt{faro}, a new package built to use the Gen3 butler, and that is part of the standard science pipeline builds. Henceforth \texttt{faro} will be the tool we use to measure data quality metrics for Rubin Observatory. Before adopting \texttt{faro}, detailed verification was performed to confirm that, given the same input datasets and configuration parameters, \texttt{faro} and \texttt{validate\_drp} produce identical results. Once this parity was demonstrated, \texttt{faro} was adopted and will now evolve (i.e., it will no longer be expected to produce identical results to those from \texttt{validate\_drp}).
+
```

Comment:
Add a statement that we ported many of the algorithms from validate_drp and have improved on them in faro and also added new metrics.   (I don't it to seem like we threw away the scientifc work of validate_drp)